class Error(Exception):
    data = {}
    name = None

    def __init__(self, *args: object, **kwargs) -> None:
        super().__init__(*args)
        self.data = kwargs.get("data")
        self.name = kwargs.get("name")



class DeliveryError(Error):
    def __str__(self):
        return "publish-in-channel-error"

